contrib/lang/erlang-elixir/packages.el

```diff
@@ -21,6 +22,41 @@
   "List of all packages to install and/or initialize. Built-in packages
 which require an initialization must be listed explicitly in the list.")
 
+(defun erlang-elixir/init-alchemist ()
+  (add-hook 'elixir-mode-hook 'alchemist-mode)
+  (setq alchemist-project-compile-when-needed t)
+  (evil-leader/set-key-for-mode 'elixir-mode
+    "mmd" 'alchemist-mix-execute
+    "mta" 'alchemist-mix-test
+    "mtb" 'alchemist-mix-test-this-buffer
+    "mtf" 'alchemist-mix-test-file
+    "mtp" 'alchemist-mix-test-at-point
+    "mtB" 'alchemist-project-open-tests-for-current-file
+    "mcc" 'alchemist-mix-compile
+    "mcb" 'alchemist-compile-this-buffer
+    "mcf" 'alchemist-compile-file
```

Comment:
Same remark as above, keep only `mcb` with an autosave (I'm adding `mcb` to the conventions).